core/state/state_object.go

```diff
@@ -519,10 +517,3 @@ func (s *stateObject) Balance() *big.Int {
 func (s *stateObject) Nonce() uint64 {
 	return s.data.Nonce
 }
-
```

Comment:
Hm, when did we stop needing this, I wonder?

Reply:
I guess long time ago? Because of various evm refactoring, etc.